apps/web/src/pages/Teams/IndexTeamPage/components/TeamTable/TeamTable.tsx

```diff
@@ -135,13 +231,26 @@ export const TeamTable = ({ teams }: TeamTableProps) => {
 };
 
 const TeamTableApi = () => {
-  const [{ data, fetching, error }] = useListTeamsQuery();
+  const [{ data: dataTeam, fetching: fetchingTeam, error: errorTeam }] =
+    useListTeamsQuery();
+  const [{ data: dataMe, fetching: fetchingMe, error: errorMe }] =
+    useMeRoleAssignmentsQuery();
+
+  const isFetching = fetchingTeam === true || fetchingMe === true;
```

Comment:
Could this be simplified as:
```suggestion
  const isFetching = fetchingTeam || fetchingMe;
```